assets/collage.css

```diff
@@ -42,6 +42,10 @@
   border: 0.1rem solid rgba(var(--color-foreground), 0.08);
 }
 
+.collage-card.collage-collection {
+  border: none;
+}
+
```

Comment:
@wiktoriaswiecicka @Oliviammarcello I didn't remove the border for the collection in the collage because it had a different class with a different border so I wanted to ensure (card--light-border)

https://screenshot.click/08-20-811m2-8h8we.mp4

Reply:
That makes sense 👍